src/Components/Scenes/SideNavigation/RequestListItem/RequestListItem.js

```diff
@@ -85,7 +86,9 @@ const RequestListItem = ({
       </ListItem>
       <Collapse in={openIconThree} timeout="auto" unmountOnExit>
         <List component="div" disablePadding>
-          {requestListItemText}
+          {currentRole.governance || currentRole.approver
+            ? requestListItemText
+            : requestListItemText.slice(1, -1)}
```

Comment:
Small nit/question - would it be more straightforward to have something like `[requestListItemText[1]]`?

Reply:
Revised now the code is more readable.